Projects/Domain/Error/Interface/Sources/DomainError.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  DomainError.swift
+//  DomainError
+//
+//  Created by JongHoon on 9/11/24.
+//
+
+import Foundation
+
+public enum DomainError: Error {
+  public enum AuthError: Error {
+    case needUpdateAppVersion
```

Comment:
```suggestion
    case invaildAppVersion
```
이런식으로 에러 case 네이밍 관리되면 어떤 의미의 에러인지 좋을 거 같아!

지금은 어떤 에러인지가 아니라 에러로 어떤 action을 하라는듯한 네이밍인 거 같아